src/panoptes/pocs/focuser/astromechanics.py

```diff
@@ -85,6 +85,10 @@ def move_to(self, position):
         Returns:
             int: focuser position following the move, in encoder units.
         """
+
+        # Subtract the near stop value from the position we want to move to
```

Comment:
Really this is adding the position of the near focus stop to the position we want to move to.